tests/codegen/cucode/scripts/functions.py

```diff
@@ -0,0 +1,57 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring
+
+#$ header function fib(int) results(int)
+def fib(n):
```

Comment:
Please avoid using header comments. These raise a warning in the 2 most recent versions of Pyccel and will stop being supported in version 2.0 (I already have a branch which has removed some of the code that handles this)

Reply:
See https://github.com/pyccel/pyccel/issues/1487

Reply:
[1fee9a6](https://github.com/pyccel/pyccel-cuda/pull/32/commits/1fee9a6d42c194d9f35da2047cf1de799b33703d)